lib/baby_squeel/compat.rb

```diff
@@ -4,6 +4,7 @@ module Compat
     # in order to behave more like Squeel
     def self.enable!
       BabySqueel::DSL.prepend BabySqueel::Compat::DSL
+      BabySqueel::Resolver.prepend BabySqueel::Compat::Resolver
```

Comment:
Rather than overriding the methods in the resolver, it would probably be simpler just to put the logic for `:table` in the resolver itself.

Reply:
Yeah it would be simpler, I put it here to keep all the compatibility related logic in one place.

Reply:
That's pretty reasonable, but I think in this case it might be better just to add it to resolver? Would you be willing to do it that way instead?

Reply:
will do.